teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -157,21 +165,24 @@ public void run() {
     }
   }
 
-  private void confirmExits() {
+  private boolean confirmExits() {
     SUB_COMMAND_LOG.display("Exits are going to be generated for validators: ");
     SUB_COMMAND_LOG.display(getValidatorAbbreviatedKeys());
     SUB_COMMAND_LOG.display("Epoch: " + epoch.toString());
     SUB_COMMAND_LOG.display("");
-    SUB_COMMAND_LOG.display(
-        "These validators won't be able to be re-activated, and withdrawals aren't likely to be possible until Phase 2 of eth2 Mainnet.");
+    SUB_COMMAND_LOG.display(WITHDRAWALS_PERMANENT_MESASGE);
+    if (!spec.isMilestoneSupported(SpecMilestone.CAPELLA)) {
+      SUB_COMMAND_LOG.display(WITHDRAWALS_NOT_ACTIVE);
+    }
     SUB_COMMAND_LOG.display("Are you sure you wish to continue (yes/no)? ");
-    Scanner scanner = new Scanner(System.in, Charset.defaultCharset().name());
+    Scanner scanner = new Scanner(System.in, Charset.defaultCharset());
```

Comment:
oh good I saw that warning after my last PR and was going to ask you to fix it. :)

Reply:
:) I figured may as well leave it cleaner than what it was since i was there.